### DISCUSSION TITLE
Spell Points System 5e

### DISCUSSION THREAD

**Discussion by Oriondeveloper8** [kgar/foundry-vtt-tidy-5e-sheets#275] [Ideas]
Want to thank you for your current work and support. 

Is there any thought about including an option to use the Variant Rule  Spell Points on your sheet instead of Spell Slots? I'm currently tracking this as a resource but having to manually update slows down encounters.

**Comment by kgar** [comment]
I'll have to see what is being planned for the dnd5e system. If a spell points option is coming soon-ish, I may want to wait so I can have system support (for data models, for example). Coincidentally, I have been thinking about it also, after seeing the Spell Points module that currently exists out there.

**Comment by kgar** [comment]
Also, thank you for the kind words. I hope to really upgrade these sheets in the coming year, after I establish a bunch of compatibility with other modules and take over the Tidy module ID in Foundry.
